warehouse/macros/payments_tests_date_spine.sql

```diff
@@ -0,0 +1,40 @@
+{% macro payments_tests_date_spine() %}
```

Comment:
I would consider making this an ephemeral model rather than a macro; IMO macros without arguments tend to be code smell